packages/flutter_tools/test/commands.shard/permeable/packages_test.dart

```diff
@@ -37,10 +38,15 @@ void main() {
       final String projectPath = await createProject(tempDir, arguments: arguments);
       final File pubspec = globals.fs.file(globals.fs.path.join(projectPath, 'pubspec.yaml'));
       String content = await pubspec.readAsString();
-      content = content.replaceFirst(
-        '\ndependencies:\n',
-        '\ndependencies:\n  $plugin:\n',
-      );
+      final List<String> contentLines = LineSplitter.split(content).toList();
+      final int depsIndex = contentLines.indexOf('dependencies:');
+      if (depsIndex != -1) {
```

Comment:
Does it pass if we expect `depsIndex != -1`?  I know that's not how it works now, but it seems more strictly correct.

Reply:
Good call. Changed to expect().